src/main/java/com/kenshoo/xlsx2csv/XlsxToCsvConverter.java

```diff
@@ -50,20 +52,36 @@ public XlsxToCsvConverter(String delimiter) {
     }
 
     /**
+     *
+     * @param delimiter - a delimiter used to differentiate values in the translated file.
+     * @param rowCacheSize - cache size of rows to save in memory (for streaming purposes).
+     * @param bufferSize - buffer size to keep in memory (for streaming purposes).
+     */
+    public XlsxToCsvConverter(String delimiter, int rowCacheSize, int bufferSize) {
```

Comment:
this is why a builder pattern is better, you have "sane" defaults for everything and user only needs to change what he whats (without a special constructor for each case).